Assets/Scripts/NPC/CrowdSpawner.cs

```diff
@@ -0,0 +1,74 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace NpcSystem.CrowdSystem
+{
+    public sealed class CrowdSpawner : MonoBehaviour
+    {
+        [SerializeField] private GameObject npcPrefab;
+        [SerializeField , Tooltip("The space between the characters")] private float spacing = 1.0f;
+        [SerializeField , Tooltip("A point to mark the start of the crowd")] private Vector3 startPoint;
+        [SerializeField , Tooltip("A Point to mark the end of the crowd")] private Vector3 endPoint;
+        [SerializeField, Tooltip("The ammount of armed characters that should spawn")] private int armedCharacterAmount;
+        [SerializeField] private bool debugGizmos;
```

Comment:
```suggestion
        [Header("References")]
        [SerializeField] private GameObject npcPrefab;
        [Header("Settings")]
        [SerializeField, Tooltip("The space between the characters")] private float spacing = 1.0f;
        [SerializeField, Tooltip("A point to mark the start of the crowd")] private Vector3 startPoint;
        [SerializeField, Tooltip("A Point to mark the end of the crowd")] private Vector3 endPoint;
        [SerializeField, Tooltip("The ammount of armed characters that should spawn")] private int armedCharacterAmount;
        [Header("Debug")]
        [SerializeField] private bool debugGizmos;
```

1. Spatie tussen `SerializeField` en `,` weg gehaald bij 3 vars.
2. Added headers.

Zou je hiervan ook een gif of video willen sturen van hoe dit werkt, als comment op ergens op discord. Ben benieuwd door de start en end points.